docs/modules/manual-testing/pages/test-in-app-camera.adoc

```diff
@@ -1,31 +1,23 @@
 = Test in-app camera
 :navtitle: Test in-app camera
 
-Learn how to test your app's camera during a manual session using image injection.
-
-image::manual-testing:image-injection-feature.jpg[Image injection feature on the left-side navigation bar,800,500]
```

Comment:
@thaobtnguyen We won't be adding any images to the intro paragraph so I removed this image tag. Also, be sure that you're only adding images to where I've left `image::$NEEDSIMAGE$` (see [documentation](https://docs.google.com/document/d/1QdGxrh0J37j0Tp-qX1BuHHhcklYDDtmKlm5x0jTmqpE/edit#)).

Reply:
@internetisaiah I saw you put the screenshot right below the opening on adb commands doc, so I just imitate. Also, I think it's rational to put the image on this place. Should you reconsider about this? cc @tungmhoang

Reply:
@thaobtnguyen It depends on the situation and the diataxis type. The doc from Isaiah has a diataxi type of reference, so use of image is limited and only provided once or twice, while this doc is of the How-to type, so we can add images under each steps.
From this analysis, I don't think we need to add an image right after the opening sentence. We can add it after each how-to step.
Come back to me and Isaiah if this make sense or you have another idea. Thank you.